internal/converter/types.go

```diff
@@ -587,21 +587,22 @@ func (c *Converter) recursiveConvertMessageType(curPkg *ProtoPackage, msgDesc *d
 	return jsonSchemaType, nil
 }
 
-func formatDescription(sl *descriptor.SourceCodeInfo_Location) string {
-	var lines []string
+func formatTitleAndDescription(sl *descriptor.SourceCodeInfo_Location) (string, string) {
+	var title, description []string
 	for _, str := range sl.GetLeadingDetachedComments() {
 		if s := strings.TrimSpace(str); s != "" {
-			lines = append(lines, s)
+			title = append(title, s)
+			description = append(description, s)
 		}
 	}
 	if s := strings.TrimSpace(sl.GetLeadingComments()); s != "" {
-		lines = append(lines, s)
+		description = append(description, s)
 	}
```

Comment:
I don't think the leading comment should be included in the description if it's already in the title.

I imagine in most cases where I would display a description, I would also display the title, so the first line would get repeated, unless I programatically cut out the first line from the description on the display side.

In, what I would imagine, a rare case where I do want to make the first line look like part of the description, I can just display the title and description in order with the same styling, which seems like much less of a hassle than cutting out the first line.